<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtlEditInfoPage</class>
 <widget class="QDialog" name="HtlEditInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_top">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit Hotel Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PlaceLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel Place</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlaceLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TelLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel Tel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="TelLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddrLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel Address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddrLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CommentLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="d_CommentLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CommentNumLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment Number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="d_CommentNumLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
